v4/source/posts.yaml

```diff
@@ -34,7 +34,7 @@
                   type: string
               props:
                 description: A general JSON property bag to attach to the post
-                type: string
+                type: JSON object
```

Comment:
It seems we don't use a "JSON object" as a type elsewhere, so please do let me know if this is an incorrect term.

That said, the "string" type isn't really correct either.

Reply:
The correct type is "object" as above.

Reply:
There we go, thanks Chris!